pkg/tests/schema_pulumi_test.go

```diff
@@ -2145,8 +2134,6 @@ Resources:
         [id=newid]
         [urn=urn:pulumi:test::test::prov:index/test:Test::mainRes]
       ~ listProps: [
-            [0]: "val1"
```

Comment:
Removing the collection from the detailed diff removes the unchanged items in it:
https://github.com/pulumi/pulumi-terraform-bridge/blob/cf59a2592ae7257786c441bf1a85bddad0246940/pkg/tfbridge/provider.go#L1159

Can be added back if we decide, will leave it away for now.

Reply:
reverting from this PR as out of scope.